scripts/install

```diff
@@ -84,7 +84,11 @@ fi
 # Build cjdns
 if ! [ -x "/opt/cjdns/cjdroute" ]; then
     here=`pwd`
-    cd /opt/cjdns && sudo Seccomp_NO=1 NO_NEON=$CJDNS_NO_NEON CFLAGS="$CJDNS_CFLAGS" ./do && cd $here
+    if [[ $RPI_REVISION == *"00"* ]]; then
```

Comment:
You know, I thought it wasn't possible to do so but I have an idea now looking back. I'll close this now and work it out!